sdk/eventhub/service.projects

```diff
@@ -13,7 +13,12 @@
 -->
 
 <Project>
-  <ItemGroup Condition="'$(SDKType)' == '' or '$(SDKType)' == 'client'">
+  <!-- Include all if no type is specified -->
+  <ItemGroup Condition="'$(SDKType)' == ''">
```

Comment:
I think this might be a little too loosely coupled as I can definitely see us defaulting SDKType to a given value higher up and that would break this assumption you trying to use here.

Reply:
Added a default all value